### PR TITLE
javascript-methods

### PR DESCRIPTION
**Pull request by Alex-piper**
Screen Shot 2022-10-19 at 10 00 19 AM (2)

### CODE REVIEW

File: javascript-methods/main.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+};
+
+var css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou'
+};
+
+library.push(js);
+library.unshift(css);
+library.splice(1);
```
Comment (Zhiwutian) [inline javascript-methods/main.js:45]: you are missing the second argument here which tells splice how many items to remove. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update

**Comment by Alex-piper** [comment]
Screen Shot 2022-10-19 at 10 07 05 AM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**